editor/connections_dialog.cpp

```diff
@@ -205,6 +208,46 @@ void ConnectDialog::_remove_bind() {
 	cdbinds->params.remove_at(idx);
 	cdbinds->notify_changed();
 }
+/*
+ * Generates 
+ */
+StringName ConnectDialog::_generate_method_callback_name(Node *p_source, String p_signal_name, Node *p_target) {
+	String node_name = p_source->get_name();
+	for (int i = 0; i < node_name.length(); i++) { // TODO: Regex filter may be cleaner.
```

Comment:
There is already the `String.validate_node_name` method. Maybe we need `String.validate_identifier` too?

Reply:
Beyond the scope of this PR. String is a core class and modifying it would also mean recompiling essentially the whole engine for everyone.